docs/rules/no-multiple-assertions-wait-for.md

```diff
@@ -30,11 +30,11 @@ Examples of **correct** code for this rule:
 const foo = async () => {
   await waitFor(() => expect(a).toEqual('a'));
   expect(b).toEqual('b');
-  
+
   // or
   await waitFor(function() {
-    expect(a).toEqual('a')
-  })
+    expect(a).toEqual('a');
+  });
```

Comment:
I know it's minor, but are we adding this for any particular reason? It's not related to this rule 🤔  but @Belco90  comment makes me thing that perhaps this branch started based off master...? (the one about the readme)

Reply:
When I did a pull from v4 to fix conflicts (due to this PR entering first #189 ) I might have used my auto format here and it fixes semicolons on `js` snippets. But indeed, I believe I started this branch from the master on my forked repo.

Reply:
But by all means if you see something wrong in these commits and needs to be fixed, let me know.

Reply:
It seems like your autoformatting is updating stuff in other files. Not a big deal, but keep an eye on it as it could lead to other issues in the future.